### ISSUE TITLE
TEES tests are failing on Travis

### ISSUE THREAD

**Issue by bgyori** [sorgerlab/indra#805]
Tees tests are failing with:
```
======================================================================
ERROR: indra.tests.test_tees.test_process_phosphorylation
----------------------------------------------------------------------
Traceback (most recent call last):
 File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
   self.test(*self.arg)
 File "/home/travis/build/sorgerlab/indra/indra/tests/test_tees.py", line 20, in test_process_phosphorylation
   tp = api.process_text(s)
 File "/home/travis/build/sorgerlab/indra/indra/sources/tees/api.py", line 89, in process_text
   python2_path)
 File "/home/travis/build/sorgerlab/indra/indra/sources/tees/api.py", line 147, in run_on_text
   if not os.path.isdir(tees_path):
 File "/home/travis/virtualenv/python3.5.6/lib/python3.5/genericpath.py", line 42, in isdir
   st = os.stat(s)
TypeError: stat: can't specify None for path argument
-------------------- >> begin captured stdout << ---------------------
Found python 2 interpreter at /opt/pyenv/shims/python2.7
```
This could be due to changes in #799 (though not obvious to me at first glance), and seems to be because this piece of code https://github.com/sorgerlab/indra/blob/master/indra/sources/tees/api.py#L116-L144 produces a None value for `tees_path`.

**Comment by bgyori** [comment]
We are now getting these errors on TEES tests:
```
======================================================================
ERROR: indra.tests.test_tees.test_process_phosphorylation
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/build/sorgerlab/indra/indra/tests/test_tees.py", line 20, in test_process_phosphorylation
    tp = api.process_text(s)
  File "/home/travis/build/sorgerlab/indra/indra/sources/tees/api.py", line 89, in process_text
    python2_path)
  File "/home/travis/build/sorgerlab/indra/indra/sources/tees/api.py", line 148, in run_on_text
    raise Exception('Provided TEES directory does not exist.')
Exception: Provided TEES directory does not exist.
-------------------- >> begin captured stdout << ---------------------
Found python 2 interpreter at /opt/pyenv/shims/python2.7
--------------------- >> end captured stdout << ----------------------
======================================================================
ERROR: indra.tests.test_tees.test_process_dephosphorylation
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/build/sorgerlab/indra/indra/tests/test_tees.py", line 52, in test_process_dephosphorylation
    tp = api.process_text(s)
  File "/home/travis/build/sorgerlab/indra/indra/sources/tees/api.py", line 89, in process_text
    python2_path)
  File "/home/travis/build/sorgerlab/indra/indra/sources/tees/api.py", line 148, in run_on_text
    raise Exception('Provided TEES directory does not exist.')
Exception: Provided TEES directory does not exist.
-------------------- >> begin captured stdout << ---------------------
Found python 2 interpreter at /opt/pyenv/shims/python2.7
--------------------- >> end captured stdout << ----------------------
======================================================================
ERROR: indra.tests.test_tees.test_process_increase_amount
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/build/sorgerlab/indra/indra/tests/test_tees.py", line 90, in test_process_increase_amount
    tp = api.process_text(s)
  File "/home/travis/build/sorgerlab/indra/indra/sources/tees/api.py", line 89, in process_text
    python2_path)
  File "/home/travis/build/sorgerlab/indra/indra/sources/tees/api.py", line 148, in run_on_text
    raise Exception('Provided TEES directory does not exist.')
Exception: Provided TEES directory does not exist.
-------------------- >> begin captured stdout << ---------------------
Found python 2 interpreter at /opt/pyenv/shims/python2.7
--------------------- >> end captured stdout << ----------------------
======================================================================
ERROR: indra.tests.test_tees.test_process_decrease_amount
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/build/sorgerlab/indra/indra/tests/test_tees.py", line 119, in test_process_decrease_amount
    tp = api.process_text(s)
  File "/home/travis/build/sorgerlab/indra/indra/sources/tees/api.py", line 89, in process_text
    python2_path)
  File "/home/travis/build/sorgerlab/indra/indra/sources/tees/api.py", line 148, in run_on_text
    raise Exception('Provided TEES directory does not exist.')
Exception: Provided TEES directory does not exist.
-------------------- >> begin captured stdout << ---------------------
Found python 2 interpreter at /opt/pyenv/shims/python2.7
--------------------- >> end captured stdout << ----------------------
======================================================================
ERROR: indra.tests.test_tees.test_process_bind
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/build/sorgerlab/indra/indra/tests/test_tees.py", line 148, in test_process_bind
    tp = api.process_text(s)
  File "/home/travis/build/sorgerlab/indra/indra/sources/tees/api.py", line 89, in process_text
    python2_path)
  File "/home/travis/build/sorgerlab/indra/indra/sources/tees/api.py", line 148, in run_on_text
    raise Exception('Provided TEES directory does not exist.')
Exception: Provided TEES directory does not exist.
-------------------- >> begin captured stdout << ---------------------
Found python 2 interpreter at /opt/pyenv/shims/python2.7
--------------------- >> end captured stdout << ----------------------
======================================================================
ERROR: indra.tests.test_tees.test_evidence_text
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/build/sorgerlab/indra/indra/tests/test_tees.py", line 186, in test_evidence_text
    tp = api.process_text(corpus)
  File "/home/travis/build/sorgerlab/indra/indra/sources/tees/api.py", line 89, in process_text
    python2_path)
  File "/home/travis/build/sorgerlab/indra/indra/sources/tees/api.py", line 148, in run_on_text
    raise Exception('Provided TEES directory does not exist.')
Exception: Provided TEES directory does not exist.
-------------------- >> begin captured stdout << ---------------------
Found python 2 interpreter at /opt/pyenv/shims/python2.7
--------------------- >> end captured stdout << ----------------------
======================================================================
ERROR: indra.tests.test_tees.test_evidence_pmid
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/build/sorgerlab/indra/indra/tests/test_tees.py", line 208, in test_evidence_pmid
    tp = api.process_text(s, pmid)
  File "/home/travis/build/sorgerlab/indra/indra/sources/tees/api.py", line 89, in process_text
    python2_path)
  File "/home/travis/build/sorgerlab/indra/indra/sources/tees/api.py", line 148, in run_on_text
    raise Exception('Provided TEES directory does not exist.')
Exception: Provided TEES directory does not exist.
-------------------- >> begin captured stdout << ---------------------
Found python 2 interpreter at /opt/pyenv/shims/python2.7
--------------------- >> end captured stdout << ----------------------
```

**Comment by bgyori** [comment]
@steppi could you please take a look at this again since there is now a new error - see above?

**Comment by steppi** [comment]
The issue is that the TEES directory is only created if run_slow is set to true but the TEES_PATH is being set regardless of whether the directory was created

**Comment by steppi** [comment]
This must have been unfixed when the later fixes for non public tests were made

**Comment by steppi** [comment]
I've submitted a PR with passing tests

**Comment by bgyori** [comment]
Fixed in e29824625b2419c1374849fa177a72843a917a81
